bootstrap/index.html

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        
+        <title>Instagram Images</title>
+        
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Inter" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Rubik" rel="stylesheet">
+
+        <style>
+
+            html, body {
+                margin: 0; 
+                height: 100%; 
+                overflow: hidden;
+                background: #0D6EFD;
+                font-family: 'Inter';
+                position: relative;
+            }
+            .header {
+                color: #FFFFFF;
+                font-size: 2.4vh;
+                font-family: 'Inter';
+                font-style: normal;
+                border-bottom: 0.1px groove white;
+                text-align:right;
+            }
+    
+            .footer {
+                position: fixed;
+                font-family: 'Inter';
+                font-style: normal;
+                bottom: 0.1vw;
+                width: 100%;
+                color: white;
+                text-align: center;
+                background-image: url("images/screenly_blur.svg");
+                mix-blend-mode:normal;
+                height: 50%;
+                
+              }
+
+            .footer-text {
+                position: fixed;
+                font-family: 'Inter';
+                font-style: normal;
+                bottom: 0.1vw;
+                width: 100%;
+                color: white;
+                text-align: center;
+            }
+
+            .instagram-post {
+                border-radius: 1vw 1vw 0vw 0vw;
+              }
+              
+            .caption {
+              background: linear-gradient(0deg, rgba(255, 255, 255, 0.99), rgba(255, 255, 255, 0.99)), #0D6EFD;
```

Comment:
Follow the same indentation throughout. I can see a mix of 2 spaces and 4 spaces.